packages/react-dom/src/client/ReactDOMHostConfig.js

```diff
@@ -247,32 +242,40 @@ export function getPublicInstance(instance: Instance): * {
   return instance;
 }
 
-export function prepareForCommit(containerInfo: Container): void {
+export function prepareForCommit(containerInfo: Container): Object | void {
   eventsEnabled = ReactBrowserEventEmitterIsEnabled();
   selectionInformation = getSelectionInformation();
+  let activeInstance;
```

Comment:
In this file, "instance" usually refers to the `Instance` type. So this is a bit confusing and easy to accidentally return an actual `Instance`, which type checker wouldn't catch.

Maybe `focusedInstanceHandle`, to match other `*InstanceHandle` in this file?